evcouplings/complex/protocol.py

```diff
@@ -22,19 +22,68 @@
 )
 from evcouplings.align.protocol import modify_alignment
 
+from evcouplings.align.alignment import Alignment
+
 from evcouplings.complex.alignment import (
     write_concatenated_alignment
 )
 from evcouplings.complex.distance import (
     find_possible_partners, best_reciprocal_matching,
-    plot_distance_distribution
+    plot_distance_distribution, get_distance
 )
 from evcouplings.complex.similarity import (
     read_species_annotation_table,
     most_similar_by_organism,
     filter_best_reciprocal,
     find_paralogs
 )
+from evcouplings.couplings.mapping import (
+    SegmentIndexMapper, Segment, segment_map_positions_single_column
+)
+
+
+def _split_annotation_string(string):
```

Comment:
I think we already have a helper function for this somewhere in the align module, please reuse if possible

Reply:
yup, align.alignment.parse_header